client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -149,10 +152,24 @@ export class DisplayTable extends React.Component {
       .map(_.last)
       .value();
 
+    const data_csv = [ordered_column_keys].concat(
+      _.map(sorted_filtered_data, (row) =>
+        _.map(ordered_column_keys, (key) =>
+          col_configs_with_defaults[key].raw_formatter
+            ? col_configs_with_defaults[key].raw_formatter(row[key])
+            : row[key]
+        )
+      )
+    );
+
     return (
       <div
         style={{ overflowX: "auto", marginTop: "20px", marginBottom: "20px" }}
       >
+        <DisplayTableUtils
+          table_name={table_name}
+          data_to_csv_string={d3.csvFormatRows(data_csv)}
```

Comment:
Small nitpick, but based on the name I'd expect `data_to_csv_string` to be a function, as it sounds like it's describing an action. Just `csv_string` feels like a clearer prop name.

Reply:
ah good point